packages/vuetify/src/locale/el.ts

```diff
@@ -0,0 +1,14 @@
+export default {
```

Comment:
had this been deleted somehow?

Reply:
No, but the iso-639-1 code for greek is `el`, not `gr`. So I added `el.ts` but left `gr.ts` for bc